apps/lib/apps.c

```diff
@@ -915,8 +916,6 @@ int load_key_certs_crls(const char *uri, int maybe_stdin,
     uidata.prompt_info = uri;
 
     if (uri == NULL) {
-        BIO *bio;
-
```

Comment:
I suppose you could also do BIO_free() immediately after the OSSL_STORE_attach().

Reply:
Done

Reply:
Now there is no empty line after declaration. Sorry for nitpicking.